api/Pipfile

```diff
@@ -52,6 +53,7 @@ gunicorn = "*"
 sentry-sdk = "*"
 django-tqdm = "*"
 limit = "*"
+fakeredis = "*"
```

Comment:
Does this need to be in both the dev dependencies and the regular ones?

Reply:
It should just be in the dev ones, whoops!